### ISSUE TITLE
write empty xml output before starting the tests to detect sefaults

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/googletest#342]
```
If you specify to generate xml outputs and use the generated files in a build 
server like hudson, problems occur if a test binary segfaults. Currently no 
output is written then and hudson has no chance to see if the test was executed 
but failed or not, because there simply is no xml file generated. The absence 
of a xml file should not be the indication for a crashed test.

A simple solution for this problem would be to first write an xml file 
indicating a failure of the test program, then execute all unit tests, and in 
the end replace the contents of the xml file with the real results. If the unit 
tests segfault in between, an error file is still left over.
```

Original issue reported on code.google.com by `johannes...@googlemail.com` on 6 Dec 2010 at 5:02
- Merged into: #311
